department-of-veterans-affairs/abd-vro
# Swagger fix

# Swagger FIx
## Description



### What was the problem?

Could not run swagger for testing since the server was hardcoded to localhost. In addition title and contact information were read from the starter project. I could not find a way to override them with application.properties on our end.

### How does this fix it?

Removes OpenAPICOnfiguration from Starter project and replicates it in abd-vro. ALl the fileds are now read from application properties (accept security). Dev and QA servers are also added for convenience.

### Jira Tickets



- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How to test this PR

Open swagger and make sure end points are working for default. Swithch to dev or qa and make sure "curls" looks good.

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
